src/pages/Integrations/Create/schema.ts

```diff
@@ -18,6 +18,7 @@ export const schema = (category) => ({
         integrationsStep(category),
         {
           title: 'Enter details',
+          label: 'Enter the details for your integration.',
```

Comment:
Oh we should move the details step to a different file since there's going to be some conditionals.